google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/ConnectionImplTest.java

```diff
@@ -620,6 +633,70 @@ public void testExecuteGetOptimizerVersion() {
     }
   }
 
+  @Test
+  public void testExecuteSetReturnCommitStats() {
+    try (ConnectionImpl subject =
+        createConnection(
+            ConnectionOptions.newBuilder()
+                .setCredentials(NoCredentials.getInstance())
+                .setUri(URI)
+                .build())) {
+      assertThat(subject.isReturnCommitStats(), is(equalTo(false)));
+
+      StatementResult res = subject.execute(Statement.of("set return_commit_stats=true"));
+      assertThat(res.getResultType(), is(equalTo(ResultType.NO_RESULT)));
+      assertThat(subject.isReturnCommitStats(), is(equalTo(true)));
+
+      res = subject.execute(Statement.of("set return_commit_stats=false"));
+      assertThat(res.getResultType(), is(equalTo(ResultType.NO_RESULT)));
+      assertThat(subject.isReturnCommitStats(), is(equalTo(false)));
```

Comment:
assertFalse is simpler and easier to read

Reply:
Done.